config/toml.go

```diff
@@ -355,7 +355,11 @@ recv-rate = {{ .P2P.RecvRate }}
 #######################################################
 [mempool]
 
-recheck = {{ .Mempool.Recheck }}
+# recheck has been moved from a config option to a global
+# consensus param in v0.36
+# See xxxx for more information.
```

Comment:
I'm not sure what should go at the xxxx here. I guess we could link to this issue for now?